pkg/machine/fcos.go

```diff
@@ -108,6 +114,30 @@ func (f FcosDownload) HasUsableCache() (bool, error) {
 	return sum.Encoded() == f.Sha256sum, nil
 }
 
+func (f FcosDownload) CleanCache() error {
+	// Set cached image to expire after 2 weeks
+	// FCOS refreshes around every 2 weeks, assume old images aren't needed
+	expire := 14 * 24 * time.Hour
+	now := time.Now()
+
+	err := filepath.Walk(f.CacheDir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
```

Comment:
What is this? This can never happen.

Reply:
I think handles failure to  access a path

Reply:
Ack but should we abort because of it? Seems like it would make sense to still continue looking for more files.